Header header

int8 system_state

int8 SYSTEM_UP = 1
int8 ROBOT_WAKEUP = 2
int8 SYSTEM_READY = 3
int8 SESSION_BEGIN = 4
int8 SESSION_END = 5
int8 ROBOT_SLEEP = 6
int8 SYSTEM_DOWN = 7
